tests/unit/Tag/TagTitleTest.php

```diff
@@ -110,6 +110,23 @@ function () {
                 Tag::appendTitle('Title');
                 
                 expect(Tag::getTitle())->equals("<title>Main - Category - Title</title>" . PHP_EOL);
+                
+                Tag::resetInput();
+                
+                Tag::setTitle('Main');
+                Tag::setTitleSeparator(' - ');
+                Tag::appendTitle(['Category', 'Title']);
+                
+                expect(Tag::getTitle())->equals("<title>Main - Category - Title</title>" . PHP_EOL);
+				
```

Comment:
Please, excuse me (and sorry), but... cant get where exactly I must put space (tab) :/